### PR TITLE
[D2IQ-64169] Docs for Elastic 3.1.1-7.6.0 upgrade

### PR DESCRIPTION
**Pull request by vibhujain**
## Jira Ticket
https://jira.mesosphere.com/browse/D2IQ-64169

Doc Team JIRA link: https://jira.d2iq.com/browse/D2IQ-64912

Universe PR: https://github.com/mesosphere/universe/pull/2486
https://github.com/mesosphere/universe/pull/2487
## Description of changes being made
Docs for Elastic basetech upgrade to 3.1.1-7.6.0

### CODE REVIEW

File: pages/mesosphere/dcos/version-policy/index.md [hunk 1]
```diff
@@ -408,7 +408,7 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka 2.9.x-2.4.0 (Recommended)</td>
+        <td>Kafka 2.8.x-2.3.1 (Recommended)</td>
```
Comment (rishabh96b) [inline pages/mesosphere/dcos/version-policy/index.md]: Why is Kafka changed here @vibhujain ?

Comment (rishabh96b) [reply pages/mesosphere/dcos/version-policy/index.md]: Oops! I saw the first commit only.

File: pages/mesosphere/dcos/version-policy/index.md [hunk 2]
```diff
@@ -366,13 +366,13 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Elastic 2.7.x-6.8.1</td>
+        <td>Elastic 3.1.0-7.4.1</td>
```
Comment (viivek46) [inline pages/mesosphere/dcos/version-policy/index.md]:
```suggestion
        <td>Elastic 2.7.x-6.8.1</td>
```

Comment (rishabh96b) [reply pages/mesosphere/dcos/version-policy/index.md]: Hmm, this is to keep the latest major version + latest major version -1 in the docs (i.e 7.X.X and 6.X.X) at any point of time.

Comment (vibhujain) [reply pages/mesosphere/dcos/version-policy/index.md]: Done. Thanks for the suggestion!

**Comment by vibhujain** [comment]
@SarahStegall Here's the Doc team JIRA link https://jira.d2iq.com/browse/D2IQ-64912
Please care to review it by Monday? It'd be wonderful. :)

**Pull request merged**